component/header.jsx

```diff
@@ -51,33 +52,35 @@ const Version = () => (
   <Div fg="comment">#/bin/nan --hash={HASH.slice(0, 80 - 17)}</Div>
 )
 
-const Nav = () => (
+const Nav = ({ path }) => (
   <nav>
     <Version />
     {'\n'}
     <Title>Menu</Title>
     {'\n'}
     <ul style={{ width: '100%' }}>
       {'  '}
-      <LinkMatch href="/" match>
+      <LinkMatch path={path} href="/">
         Home
       </LinkMatch>
       {' - '}
-      <LinkMatch href="/doc">Documentation</LinkMatch>
+      <LinkMatch path={path} href="/profile">Profile</LinkMatch>
       <LogAction />
     </ul>
     {'\n'}
   </nav>
 )
 
-export const Header = ({ user, page, title, children }) => (
+export const Header = ({ user, page, title, children }) => {
+  const { pathname: path } = useURL()
```

Comment:
this is an example of the use of the `useURL` hook, it returns a [`new URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) object of `window.location.href` and is updated everytime it changes